optuna_dashboard/static/components/GraphSlice.tsx

```diff
@@ -171,6 +188,10 @@ const plotSlice = (
       linewidth: 5,
       gridcolor: "#f2f5fa",
       gridwidth: 1,
+      tickfont: {
+        color: "#000000",
+      },
+      automargin: true, // Otherwise the label is outside of the plot
```

Comment:
Thank you for adding this option. Looks good but I'm wondering why the label of xAxis is hidden when changed xAxis. Can you find the cause of this problem?

https://user-images.githubusercontent.com/5564044/115712873-4ad55a80-a3b0-11eb-97ab-2d3fdc6a9f06.mp4

Reply:
This does not happen for me. Which browser are you using? I am using Chrome 90.0 on Windows (But server runs on Ubuntu)

Maybe it is due to a different database? Could you share the datebase with me so that I can look into it?

Reply:
Hmm... I see.

> Maybe it is due to a different database? Could you share the datebase with me so that I can look into it?

Of course, sure. I checked your pull request on the following program.

```python
import optuna
from optuna_dashboard.app import create_app


host = "127.0.0.1"
port = 8080


def create_optuna_storage() -> optuna.storages.InMemoryStorage:
    storage = optuna.storages.InMemoryStorage()

    # Single-objective study
    study = optuna.create_study(study_name="single-objective", storage=storage)

    def objective_single(trial: optuna.Trial) -> float:
        x1 = trial.suggest_float("x1", 0, 10)
        x2 = trial.suggest_float("x2", 0, 10)
        return (x1 - 2) ** 2 + (x2 - 5) ** 2

    study.optimize(objective_single, n_trials=100)
    return storage


def main() -> None:
    storage = create_optuna_storage()
    app = create_app(storage)
    app.run(host=host, port=port, reloader=True)


if __name__ == "__main__":
    main()
```

Reply:
Thank you. So far I was using sqlite as a storage. I can't find an option in optuna to read from a in-memory storage. How did you do that?

Reply:
wait, I am stupid...

Reply:
Thank you. With this data I can reproduce the bug.

Reply:
Fun fact: This bug was caused by the line that I was asking about in the other comment. Removing `xaxis: selected` solved it. I don't really know what this attribute was used for, but it looks like everything works now. I'll push the change in a second